starter-code/app.js

```diff
@@ -1,5 +1,43 @@
-const express = require('express');
+const express = require("express");
 const app = express();
-const Chuck  = require('chucknorris-io');
+const Chuck = require("chucknorris-io");
 const client = new Chuck();
 
+app.set("views", __dirname + "/views");
+app.set("view engine", "ejs");
+
+app.get("/random", (req, res, next) => {
+  // Retrieve a random chuck joke
+  client
+    .getRandomJoke()
+    .then(joke => {
+      res.render("index", joke);
```

Comment:
Is better to pass an object to the render method. There are several ways to do that. You can define that variable before or you can write:
```
res.render('index', {joke: joke.value});
```
In this case, the first "joke" is the name of the property (you can call it whatever you want). The second "joke" is the response of getRandomJoke. "value" is a property of that response and the only thing you need to pass to the render method.